src/components/GraphiQL.js

```diff
@@ -171,6 +171,7 @@ export class GraphiQL extends React.Component {
       variableEditorHeight: this._storageGet('variableEditorHeight') || 200,
       docsOpen: false,
       docsWidth: this._storageGet('docExplorerWidth') || 350,
+      showSpinner: false,
```

Comment:
I'd recommend calling this `isWaitingForResponse` since thats the semantic value, and the spinner is just the visualization of this value.